configs/cyberagent--calm2-7b-chat.json

```diff
@@ -3,7 +3,7 @@
   "model_name_or_path": "cyberagent/calm2-7b-chat",
   "lora_model_name_or_path": null,
   "tokenizer_name_or_path": null,
-  "prompt_template": "USER: {instruction}\nASSISTANT: ",
+  "prompt_template": "USER:{instruction} ASSISTANT:",
```

Comment:
Why did you make this change? I followed [the official instructions](https://huggingface.co/cyberagent/calm2-7b-chat#usage).

Reply:
> Why did you make this change? I followed [the official instructions](https://huggingface.co/cyberagent/calm2-7b-chat#usage).

Oh, it is a mistake I made. I think this change is unnecessary.